projects/libpsl/build.sh

```diff
@@ -21,60 +21,81 @@ export CPPFLAGS="-I$DEPS_PATH/include"
 export LDFLAGS="-L$DEPS_PATH/lib"
 
 cd $SRC/icu/source
-./configure --disable-shared --enable-static --disable-extras --disable-icuio --disable-layoutex \
-  --disable-tests --disable-samples --with-data-packaging=static --prefix=$DEPS_PATH
+if test $SANITIZER = "undefined"; then
```

Comment:
Let's remove this `if` statement. We should not rely on env variables other than described in https://github.com/google/oss-fuzz/blob/master/docs/new_project_guide.md#buildsh-script-environment

It's safe to always add `-fno-sanitize=` flags to CFLAGS and others even for non-UBSan configs.